gammapy/estimators/points/tests/test_sed.py

```diff
@@ -656,25 +656,25 @@ def test_flux_points_parallel_ray_actor_map(fpe_map_pwl):
     assert_allclose(actual, [0.343332, 3.162278, 29.126327], rtol=1e-5)
 
     actual = table["norm"].data
-    assert_allclose(actual, [0.974726, 0.96342, 0.994251], rtol=1e-2)
+    assert_allclose(actual, [0.95551238, 0.96532761, 0.9952374], rtol=1e-2)
 
     actual = table["norm_err"].data
-    assert_allclose(actual, [0.067637, 0.052022, 0.087059], rtol=3e-2)
+    assert_allclose(actual, [0.08350122, 0.05249954, 0.08750922], rtol=3e-2)
```

Comment:
There is a rather large changes here... And the test are also here to point out issues.
What cause this change? Do we understand it? Are they due to a subtle bug introduction?

Reply:
I am not entirely sure, but it appears to be something with the DatasetsActor because other tests utilising the same datasets methods are not failing.
@QRemy, do you have an idea?